### PR TITLE
feat(releaseRunbook): Add a bunch of UX improvements

### PR DESCRIPTION
**Pull request by kevinawoo**

### CODE REVIEW

**Comment by kevinawoo** [comment]
Hey @plumpy, @maggieneterval he's some UX/workflow improvements I found when I was going through the release manager runbook. Would you guys be able to give me an 👀 . 🙏 🙏

File: community/releases/release-manager-runbook/index.md
Comment (maggieneterval) [inline community/releases/release-manager-runbook/index.md]: I'd prefer that buildcop-related troubleshooting not be duplicated here since we already have a section on resolving Bintray 409s: https://spinnaker.io/community/contributing/nightly-builds/#bintray-conflicts

File: community/releases/release-manager-runbook/index.md
Comment (maggieneterval) [inline community/releases/release-manager-runbook/index.md:47]: What's the justification for the extra newlines in between sections? I think elsewhere we just have one newline above/below section headers, but could be wrong 🤷

File: community/releases/release-manager-runbook/index.md
Comment (maggieneterval) [inline community/releases/release-manager-runbook/index.md]:
```suggestion
    The release-manager@spinnaker.io group will have access to the
```

File: community/releases/release-manager-runbook/index.md
Comment (maggieneterval) [inline community/releases/release-manager-runbook/index.md]: If we're going to include the cherry-pick search link as this fill-in-the-blank URL, let's clarify that release managers will need to construct three separate queries, one for each supported $RELEASE_BRANCH.

**Review by maggieneterval — APPROVED** [review]
Thanks so much for taking the time to do this! ❤️ LGTM after comments addressed & @dorbin approval.

File: community/releases/release-manager-runbook/index.md
Comment (dorbin) [inline community/releases/release-manager-runbook/index.md]:
```suggestion
- [This is the expected release cadence](/community/releases/release-cadence/).
```

File: community/releases/release-manager-runbook/index.md
Comment (dorbin) [inline community/releases/release-manager-runbook/index.md]: Not sure what these items are trying to do. Are you saying "verify you've been added into the spinnaker-announce@..."?
And "Verify you've have access to the Jenkins UI"?

If so, I would precede the list with:
"Verify you have access to the following:

- You're a member of the release-manager@spinnaker.io group and the spinnaker-announce@googlegroups.com group. Contact a member of the TOC..."
- You have access to the Jenkins UI
- You can SSH into Jenkins

...and so on.

File: community/releases/release-manager-runbook/index.md
Comment (dorbin) [reply community/releases/release-manager-runbook/index.md:47]: Yeah, we try not to include superfluous newlines and spaces. 

It's harmless though. Both our current md interpreter, Kramdown, and the one we'll probably be using soon, Goldmark, ignore extra newlines.

File: community/releases/release-manager-runbook/index.md
Comment (dorbin) [inline community/releases/release-manager-runbook/index.md]:
```suggestion
job, which creates `latest-unvalidated` when it passes:
```

File: community/releases/release-manager-runbook/index.md
Comment (dorbin) [inline community/releases/release-manager-runbook/index.md]:
```suggestion
    - you can check the status of each service build by going to [GCP cloudbuild](https://console.cloud.google.com/cloud-build/builds?project=spinnaker-community). Turn on the tags to see each service name
```

Comment (dorbin) [reply community/releases/release-manager-runbook/index.md]: I'm not sure what you're trying to say here.

File: community/releases/release-manager-runbook/index.md
Comment (dorbin) [inline community/releases/release-manager-runbook/index.md:75]: How is this going to render? I try to preview it, and it looks like everything in the < details > element is ignored.

File: community/releases/release-manager-runbook/index.md
Comment (dorbin) [inline community/releases/release-manager-runbook/index.md]:
```suggestion
    1. Copy the direct link to the changelog for this version by searching for the release branch. For example: `release-1.21.x`.
```

File: community/releases/release-manager-runbook/index.md
Comment (dorbin) [inline community/releases/release-manager-runbook/index.md]: On the one hand, this is better because it shows the whole message. But on the other hand, the intro sentence makes less sense because it tries to introduce just the suggested command, not the informational message. (That is, you wouldn't "run" `You are welcome to test...`)

File: community/releases/release-manager-runbook/index.md
Comment (dorbin) [reply community/releases/release-manager-runbook/index.md]: s/will have/has/

File: community/releases/release-manager-runbook/index.md
Comment (dorbin) [inline community/releases/release-manager-runbook/index.md]:
```suggestion
If you're getting any errors that look like:
```

**Review by dorbin — CHANGES_REQUESTED** [review]

File: community/releases/release-manager-runbook/index.md
Comment (kevinawoo) [reply community/releases/release-manager-runbook/index.md]: oh cool, i didn't know about that! I added a note to the top of the runbook!

File: community/releases/release-manager-runbook/index.md
Comment (kevinawoo) [reply community/releases/release-manager-runbook/index.md:47]: For readability, since inside each section, they usually use 1 newline.

It gives some breathing room to the large document.

File: community/releases/release-manager-runbook/index.md
Comment (kevinawoo) [reply community/releases/release-manager-runbook/index.md:75]: So meta. A gif within a gif XD
<kbd></kbd>

File: community/releases/release-manager-runbook/index.md
Comment (kevinawoo) [inline community/releases/release-manager-runbook/index.md:122]: fixed a 🐛 
`validated` should have been `unvalidated` 😄

**Comment by kevinawoo** [comment]
hey @maggieneterval @dorbin I made the changes!

I think remaining open comment is about adding the whitespace.

I added it to improve the readability of the doc, since each section uses 1 newline internally, and sections can be broken up by 2 newlines.

This gives the doc a bit more breathing room and not just a wall of text.

Thoughts?

File: community/releases/release-manager-runbook/index.md
Comment (dorbin) [inline community/releases/release-manager-runbook/index.md]: s/encouter/encoutered/

File: community/releases/release-manager-runbook/index.md
Comment (dorbin) [inline community/releases/release-manager-runbook/index.md]:
```suggestion
and [spinnaker-announce@googlegroups.com](mailto:spinnaker-announce@googlegroups.com) groups.
```

File: community/releases/release-manager-runbook/index.md
Comment (dorbin) [inline community/releases/release-manager-runbook/index.md]:
```suggestion
Contact a member of the TOC or SC to get access.
```

File: community/releases/release-manager-runbook/index.md
Comment (dorbin) [inline community/releases/release-manager-runbook/index.md]:
```suggestion
    - If the builds fail, [Google Cloud Build](https://console.cloud.google.com/cloud-build/builds?project=spinnaker-community) is a helpful UI. 
```

File: community/releases/release-manager-runbook/index.md
Comment (dorbin) [reply community/releases/release-manager-runbook/index.md:75]: 💥

File: community/releases/release-manager-runbook/index.md
Comment (dorbin) [inline community/releases/release-manager-runbook/index.md]:
```suggestion
        <summary>Click to expand a GIF of how to view the tags</summary>
```

File: community/releases/release-manager-runbook/index.md
Comment (plumpy) [inline community/releases/release-manager-runbook/index.md]:
```suggestion
- You're a member of the [release-managers@spinnaker.io](https://groups.google.com/a/spinnaker.io/forum/#!forum/release-managers)
group and a *manager* of the [spinnaker-announce@googlegroups.com](https://groups.google.com/forum/#!forum/spinnaker-announce)
group. (You'll get a permissions error on those pages if you don't have access.
```

File: community/releases/release-manager-runbook/index.md
Comment (plumpy) [inline community/releases/release-manager-runbook/index.md]:
```suggestion
- You're a member of the [release-managers GitHub team](https://github.com/orgs/spinnaker/teams/release-managers).
- You can access the [Jenkins UI](https://builds.spinnaker.io/) and you're able
to run a job. Access is controlled by the `release-managers` GitHub team, but it
may take some time for the permissions to propagate from GitHub to Jenkins.
```

File: community/releases/release-manager-runbook/index.md
Comment (plumpy) [inline community/releases/release-manager-runbook/index.md:21]:
```suggestion
## Verify You Have Access To The Following

If you don't have access to any of the following, contact a member of the TOC or SC.

```

**Review by plumpy — APPROVED** [review]
Awesome, thanks for making this better!! :dango:

**Comment by kevinawoo** [comment]
@dorbin I think I fixed everything up.

May I have 1 more review pretty please?

**Review by dorbin — APPROVED** [review]
You got it!

**Pull request merged**
